_tuning-your-cluster/availability-and-recovery/workload-management/create-workload-group-rules-api.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: default
+title: Create workload group rules
+nav_order: 20
+parent: Workload management
+grand_parent: Availability and recovery
+---
+
+# Create workload group rules
+
+The Create Workload Group Rules API for the `workload_group` feature type allows you to define, update, retrieve, and delete rules that automatically assign workload group IDs to incoming queries. When a query matches the attributes specified in a rule, OpenSearch tags the query with the corresponding workload group ID. This eliminates the need for clients to manually include the workload group ID in each request.
+
+For more information about how to use the Rule Lifecycle API, see [this page]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/rule-based-autotagging/rule-lifecycle-api/).
```

Comment:
We haven't introduced this API. Did we mean to reference the API in the preceding paragraph? Also, use the actual name of the page rather than "this page".